src/bentoml/_internal/yatai_rest_api_client/config.py

```diff
@@ -108,3 +112,16 @@ def get_current_context() -> YataiClientContext:
 def get_current_yatai_rest_api_client() -> YataiRESTApiClient:
     ctx = get_current_context()
     return ctx.get_yatai_rest_api_client()
```

Comment:
@ssheng thank you again for speedy work on this! It is indeed what we expect. 

Just a slight minder, depends how restrict your code space is, the highlighted function will be redundant after your new implementation, I think.

Reply:
Thanks for the suggestion, @ShaohongBai.